cmd/commands/runtime.go

```diff
@@ -868,7 +877,8 @@ func installComponents(ctx context.Context, opts *RuntimeInstallOptions, rt *run
 				version:      "v1alpha1",
 			},
 		},
-		saName: store.Get().RolloutReporterServiceAccount,
+		saName:     store.Get().RolloutReporterServiceAccount,
+		IsInternal: util.StringIndexOf(store.Get().CFInternalReporters, store.Get().RolloutReporterName) > -1,
```

Comment:
I don't understand this. it worked without it, why do we need this?

Reply:
I don't think it worked without it, it was one of the things I added, in order to include the label in the createReporter func

Reply:
what I don't understand is - which reporter is not internal? I thin this should be assumed that all of them are and just set it to true inside `createReporter` func